sesman/chansrv/clipboard.c

```diff
@@ -185,38 +185,38 @@ x-special/gnome-copied-files
 #define LOG_LEVEL   LOG_ERROR
 
 #define log_error(_params...)                           \
-{                                                       \
-    g_write("[%10.10u]: CLIPBOARD  %s: %d : ERROR: ",   \
-            g_time3(), __func__, __LINE__);             \
-    g_writeln (_params);                                \
-}
+    {                                                       \
+        g_write("[%10.10u]: CLIPBOARD  %s: %d : ERROR: ",   \
+                g_time3(), __func__, __LINE__);             \
+        g_writeln (_params);                                \
+    }
```

Comment:
Thank you for reformatting in a separate commit. It helps me to see the true diff.

Reply:
Tried to make review as easy as possible.